src/server/log.js

```diff
@@ -31,6 +32,7 @@ class Logger {
 
 	constructor(logFilePath) {
 		this.level = LogLevel.INFO;
+		this.emailLevel = LogLevel.ERROR;
 		this.logToFile = true;
 		this.logToConsole = true;
 		this.logFile = logFilePath;
```

Comment:
Not used

Reply:
I think we used it in this line `if (level.priority <= this.emailLevel.priority)`